.circleci/check_mysql_connection.sh

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env bash
+set -e
 
-[ -n "$MYSQL_USER" ] || MYSQL_USER=test
-[ -n "$MYSQL_PASSWORD" ] || MYSQL_PASSWORD=test
+TEST_DB_HOST="${TEST_DB_HOST:-127.0.0.1}"
```

Comment:
I guess a variable for port can be added as well. Tests use `TEST_DB_PORT` for that.

I know, we use default ports for PostgreSQL and MySQL, but for the sake of completeness...

Reply:
Yes, I agree, it's worth to add an argument for the DB port.